app/models/form526_submission.rb

```diff
@@ -468,22 +466,11 @@ def last_remediation
   end
 
   def account
-    # first, check for an ICN on the UserAccount associated to the submission, return it if found
-    account = user_account
-    return account if account&.icn.present?
+    return user_account if user_account&.icn.present?
 
-    # next, check past submissions for different UserAccounts that might have ICNs
-    past_submissions = get_past_submissions
-    account = find_user_account_with_icn(past_submissions, 'past submissions')
-    return account if account.present? && account.icn.present?
-
-    # next, check for any historical UserAccounts for that user which might have an ICN
-    user_verifications = get_user_verifications
-    account = find_user_account_with_icn(user_verifications, 'user verifications')
-    return account if account.present? && account.icn.present?
-
-    # failing all the above, default to an Account lookup
-    Account.lookup_by_user_uuid(user_uuid)
+    Rails.logger.info("Form526Submission::account - no UserAccount ICN found for user #{user_uuid}")
+    # query MPI by EDIPI first & attributes second for user ICN
+    query_mpi_account
```

Comment:
can we call this something like `get_icn_from_mpi` or something that makes it more clear what this function returns? Since this is a large file I want to make sure all the functions we add are very clear